models/User.js

```diff
@@ -0,0 +1,83 @@
+const mongoose = require('mongoose');
+const bcrypt = require('bcrypt');
+const { loginSchema, registrationSchema } = require('../utils/validations/schemas/user');
+const SALT_WORK_FACTOR = 10;
+const Schema = mongoose.Schema;
+
+const UserSchema = new Schema({
+  first_name: {
+    type: String,
+    required: true,
+    maxlength: 30,
+    trim: true,
+  },
+  last_name: {
+    type: String,
+    required: true,
+    maxlength: 30,
+    trim: true,
+  },
+  email: {
+    type: String,
+    lowercase: true,
+    trim: true,
+    unique: true,
+    required: [true, 'Email is required'],
+  },
+  username: {
+    type: String,
+    maxlength: 30,
+    trim: true,
+  },
+  password: {
+    type: String,
+    required: true,
+    trim: true,
+    minlength: 8,
+  },
+  age: {
+    type: Number,
+    required: true,
+    trim: true,
+  },
+  gender: {
+    type: String,
+    trim: true,
+  },
+  token: {
+    type: String,
+    default: '',
+  },
+  isVerified: {
+    type: Boolean,
+    default: false,
+  },
+  resetLink: {
+    data: String,
```

Comment:
what is data:String?
it should be type:String

Reply:
Mistake too, fixed already